test/incrParse/extend-identifier-at-eof.swift

```diff
@@ -0,0 +1,4 @@
+// RUN: %incr-transfer-tree --expected-incremental-syntax-tree %S/Outputs/extend-identifier-at-eof.json %s
+
+func foo() {}
+_ = x<<<|||x>>>
```

Comment:
I would add comment to this file explaining that no trailing newline should be added to the end of it.